shopinvader_wishlist/tests/test_product_set.py

```diff
@@ -69,11 +69,13 @@ def test_get_line_by_product(self):
         )
         variant = prod.shopinvader_bind_ids[0]
         self.assertEqual(
-            self.prod_set.get_line_by_product(invader_variant_id=variant.id),
+            self.prod_set.get_lines_by_product(
+                invader_variant_ids=variant.ids
+            ),
             line,
         )
         self.assertEqual(
-            self.prod_set.get_line_by_product(product_id=prod.id), line
+            self.prod_set.get_lines_by_product(product_ids=prod.ids), line
         )
         with self.assertRaises(exceptions.ValidationError):
-            self.prod_set.get_line_by_product()
+            self.prod_set.get_lines_by_product()
```

Comment:
Should we let the previous test to ensure the backward compatibility does not break in future developments?

Reply:
I'll add a test for it